api/agent/drivers/docker/docker.go

```diff
@@ -19,9 +19,15 @@ import (
 	"github.com/fnproject/fn/api/common"
 	"github.com/fnproject/fn/api/models"
 	"github.com/fsouza/go-dockerclient"
+	"github.com/rs/xid"
```

Comment:
is this necessary? we have `github.com/fnproject/fn/api/id` which generates ids based on machine id, time and a monotonic (no process id but similar to xid, but are we worried about sub millisecond restarts?) - adding dependencies is a whole thing now, and this one just seems unnecessary without any explanation

Reply:
in any event, if we are, it seems like pid + timestamp would work for this purpose fine without needing the dependency (worst case we say that machine id and monotonic don't add any entropy from xid anyway)

Reply:
Swapped this with crypto/rand implementation.